usr/lib/cca_stdll/cca_specific.c

```diff
@@ -296,13 +329,7 @@ static CK_RV init_cca_adapter_lock(STDLL_TokData_t *tokdata)
     if (cnt > 1)
         return CKR_OK;
 
-    if (pthread_rwlockattr_init(&attr) != 0) {
-        TRACE_ERROR("pthread_rwlockattr_init failed\n");
-        OCK_SYSLOG(LOG_ERR, "%s: Failed to initialize the CCA adapter lock\n",
-                   __func__);
-        return CKR_CANT_LOCK;
-    }
-
+#if !defined(_AIX)
```

Comment:
Please leave the attr initialization here (inside the #if AIX), but set the attr variable to NULL where it is declared.